### DISCUSSION TITLE
Clicking or dragging on a Control Panel over the map propagates the event to the map

### DISCUSSION THREAD

**Discussion by enelesmai** [uber/react-map-gl#1443] [Q&A]
I implemented a Slider Element inside a Control Panel over the map, the issue I have with this is that when I click or drag the slider, this event is being propagated also to the map. And the map moves at the same time I'm changing the slider position. 

How could I prevent this from happening?
I already tried using stopPropagation in the onChange event of the slider but it keeps moving the map.
I'm attaching a short video for reference.

![ezgif-4-f98bc788b02a](https://user-images.githubusercontent.com/76240324/115587162-f7211f00-a292-11eb-8db4-c11e389236b7.gif)

**Comment by Pessimistress** [comment]
You should render the panel outside of the map component, see [example](https://visgl.github.io/react-map-gl/examples/heatmap)
